syntax = "proto3";

package Proto;

enum MsgId {
    None = 0;
    MI_TestMsg = 1;

    C2L_AccountCheck = 1000;
    C2L_AccountCheckRs = 1001;
    MI_AccountCheckToHttpRs = 1005;
}